### PR TITLE
make: remove the hack script

### PR DESCRIPTION
**Pull request by jhrozek**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Since commit 6c2f361029f91edbd2ecc2766afed67190bef12f that fixed the
YAML separators, this hack is not needed anymore.

#### Which issue(s) this PR fixes:
None

#### Does this PR have test?
Tested by the existing OLM tests.

#### Special notes for your reviewer:
N/A

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -476,7 +476,7 @@ bundle: operator-sdk deployments ## Generate bundle manifests and metadata, then
 	# For some reason, the profile CM is not added automatically to the
 	# bundle, but in general bundles can contain CMs, so let's add it
 	# manually
-	python hack/extract-cm.py $(BUNDLE_OPERATOR_MANIFEST) configMap security-profiles-operator-profile > ./bundle/manifests/security-profiles-operator-profile_v1_configmap.yaml
+	#python hack/extract-cm.py $(BUNDLE_OPERATOR_MANIFEST) configMap security-profiles-operator-profile > ./bundle/manifests/security-profiles-operator-profile_v1_configmap.yaml
```
Comment (JAORMX) [inline Makefile:479]: can we just delete this line and the above comment?

**Review by JAORMX — CHANGES_REQUESTED** [review]
woo-hoo! code deletion!

File: Makefile (hunk 1 above)
Comment (jhrozek) [reply Makefile:479]: oops lol

Comment (jhrozek) [reply Makefile]: thanks, done

**Comment by jhrozek** [comment]
huh that verify change looks unrelated and weird
/test pull-security-profiles-operator-verify

**Review by saschagrunert — APPROVED** [review]
Thank you!

**Pull request merged**